src/components/Sidebar/Sidebar.tsx

```diff
@@ -103,29 +102,33 @@ export default function Sidebar(props: {
   );
 }
 
-const tabBarCss = css`
+const tabBarCss = (t: BifrostTheme) => css`
   ul {
     display: flex;
+    justify-content: space-around;
+    align-items: center;
     list-style: none;
     padding: 0;
     margin: 0;
-    justify-content: space-between;
+    padding: 8px 30px;
   }
   li {
-    padding: 15px 30px;
-    border-top-left-radius: 15px;
-    border-top-right-radius: 15px;
-    background: #e9e9e9;
+    background: transparent;
     cursor: pointer;
-
-    border: 1px solid #d4d4d4;
     border-bottom: none;
-    width: calc(100% / 3);
-    text-align: center;
+    transform: scale(1);
+    transition: transform 0.5s;
+    padding: 5px 20px;
+    border-radius: 15px;
```

Comment:
Maybe change this to 7px in order to have a square with rounded corder as in Angela's mockup?
<img width="340" alt="Screen Shot 2021-08-17 at 4 32 57 PM" src="https://user-images.githubusercontent.com/15991814/129813855-be732e44-7c28-4ca8-8f90-209d0df438d1.png">